pipeline/dwi_masking.py

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env python
 
 from __future__ import division
+
 # -----------------------------------------------------------------
-# Author:       Senthil Palanivelu, Tashrif Billah                 
-# Written:      01/22/2020                             
-# Last Updated:     02/28/2020
+# Author:           Senthil Palanivelu, Tashrif Billah
+# Written:          01/22/2020
+# Updatad by:       Ryan Zurrin
```

Comment:
Remove this and add it to `Author:` above. Your update is already recorded in GitHub.